### PR TITLE
Check for service enablement before bucket creation

### PR DESCRIPTION
**Pull request by joaopalet**
- Also fixes the check in SKE and uniformizes the approaches

### CODE REVIEW

File: internal/cmd/ske/cluster/create/create.go [hunk 1]
```diff
@@ -87,13 +87,15 @@ func NewCmd(p *print.Printer) *cobra.Command {
...
-			// Check if SKE is enabled for this project
+			// Check if the project is enabled before trying to create
 			enabled, err := skeUtils.ProjectEnabled(ctx, apiClient, model.ProjectId)
 			if err != nil {
 				return err
 			}
 			if !enabled {
-				return fmt.Errorf("SKE isn't enabled for this project, please run 'stackit ske enable'")
+				return &errors.ServiceDisabledError{
+					Service: "ske",
```
Comment (DiogoFerrao) [inline internal/cmd/ske/cluster/create/create.go:97]:
```suggestion
					Service: "SKE",
```

Comment (DiogoFerrao) [reply internal/cmd/ske/cluster/create/create.go:97]: I think we are using the capitalized `SKE` everywhere

File: internal/pkg/services/object-storage/utils/utils.go [hunk 2]
```diff
@@ -3,15 +3,33 @@ package utils
...
+		if !ok {
+			return false, err
+		}
+		if oapiErr.StatusCode == http.StatusNotFound {
+			return false, nil
+		}
+		return false, err
+	}
+	return true, nil
+}
```
Comment (DiogoFerrao) [inline internal/pkg/services/object-storage/utils/utils.go:31]: Great idea 💡

File: internal/pkg/errors/errors.go [hunk 3]
```diff
@@ -133,6 +133,11 @@ The profile name can only contain lowercase letters, numbers, and "-" and cannot
 
 	USAGE_TIP = `For usage help, run:
   $ %s --help`
+
+	SERVICE_DISABLED_ERROR = `This service isn't enabled for the current project.
+
```
Comment (DiogoFerrao) [inline internal/pkg/errors/errors.go:138]:
```suggestion
	SERVICE_DISABLED = `This service isn't enabled for the current project.

```

File: internal/cmd/ske/cluster/create/create.go (hunk 1 above)
Comment (joaopalet) [reply internal/cmd/ske/cluster/create/create.go:97]: This is just used for the command suggestion to enable (as `stackit %s enable`), similarly to what we do in other errors

**Review by DiogoFerrao — APPROVED** [review]

**Pull request merged**